blues/data/README.md

```diff
@@ -0,0 +1,3 @@
+Files used for unit testing purposes.
+circle.xml is a force field.
+squareB2 is a simple square model system.
```

Comment:
(So, when you write things, you have to try to imagine yourself as a user who knows nothing about this project and you are trying to make sense of the files and how everything fits together from only the README information provided.)